templates/store/user-details.html

```diff
@@ -0,0 +1,110 @@
+{% extends "_layout.html" %}
+
+{% block title %}{{ context.username }}{% endblock %}
+
+{% block content %}
+
+<div class="p-strip--light is-bordered is-shallow" data-js="sticky-header">
+  <header class="row">
+    <div class="p-media-object--large u-no-margin--bottom">
+      <div class="p-media-object__details">
+        <h1 itemprop="name" class="p-media-object__title u-no-margin--bottom u-sv2" style="text-transform:capitalize">
+          <span>
+            {{ context.username }}
+          </span>
+        </h1>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item">
+            <a href="#bundles">{{ context.bundles_count }} bundle{{ context.bundles_count|pluralize }}</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="#charms">{{ context.charms_count }} charm{{ context.charms_count|pluralize }}</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </header>
+</div>
+
+{% if context.bundles %}
+   <div class="p-strip">
+     <div class="row">
+       <div class="col-12">
+         <h3 id="bundles" class="profile-title">
```

Comment:
`profile-title` doesn't do anything, it can be removed.

Reply:
Done.